config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_alloydbclusters.alloydb.cnrm.cloud.google.com.yaml

```diff
@@ -178,6 +178,10 @@ spec:
                     - startTimes
                     type: object
                 type: object
+              clusterType:
+                description: 'The type of cluster. If not set, defaults to PRIMARY.
```

Comment:
We discussed in chat.  AIUI:

PRIMARY -> SECONDARY is not allowed.

SECONDARY -> PRIMARY is allowed, but it does not "replace" the existing PRIMARY.  Rather, we start a "fork" of the database, with this cluster as the PRIMARY.  So the data from the old PRIMARY and this new PRIMARY would be expected to diverge.

Reply:
(If so, I think this is fine - that behaviour shouldn't cause any problems for usage with e.g. gitops)

Reply:
Yes. Thanks for adding the discussion here